include/openmc/particle.h

```diff
@@ -169,6 +169,7 @@ class Particle {
     double E;
     double wgt;
     int delayed_group;
+    int surf_num;
```

Comment:
Since the surface ID is used, I would change this to
```suggestion
    int surf_id;
```